.github/cloud-samples-tools/pkg/config/utils.go

```diff
@@ -17,10 +17,28 @@
 package config
 
 import (
+	"encoding/json"
 	"errors"
 	"os"
+	"regexp"
 )
 
+var multiLineCommentsRegex = regexp.MustCompile(`(?s)\s*/\*.*?\*/`)
```

Comment:
nit: Go style prefers shorter variable names, relying on the context for additional information. 

I would name this just `multiLine` and the next one `singleLine`.